### DISCUSSION TITLE
Update Monterey

### DISCUSSION THREAD

**Discussion by n2ng-dev** [SkyrilHD/Dell-E7250-Hackintosh#36] [General]
The Last version can be use to update Monterey ?

**Comment by SkyrilHD** [comment]
You can use Monterey with the EFI. The only thing that do not work are WiFi and Bluetooth. WiFi can be fixed by changing AirportItlwm with Monterey compatible one (Download from [here](https://github.com/OpenIntelWireless/itlwm/releases/tag/v2.0.0)). Bluetooth is a bit harder. First, you need to upgrade [IntelBluetooth](https://github.com/OpenIntelWireless/IntelBluetoothFirmware/releases/tag/v2.0.1) and remove IntelBluetoothInjector then add BlueToolFixup which can be found in [BrcmPatchRAM repo](https://github.com/acidanthera/BrcmPatchRAM/releases/tag/2.6.0). Guide for Bluetooth can be found [here](https://openintelwireless.github.io/IntelBluetoothFirmware/FAQ.html#what-additional-steps-should-i-do-to-make-bluetooth-work-on-macos-monterey)

I will add the necessary changes when Monterey gets released. But for now, the support is on Big Sur.

**Reply by n2ng-dev** [reply]
I was change wifi card with DW1560, wifi and Bluetooth no need kext
